homeassistant/helpers/config_validation.py

```diff
@@ -349,6 +349,11 @@ def positive_timedelta(value: timedelta) -> timedelta:
     return value
 
 
+def remove_falsy(value: Sequence[T]) -> Sequence[T]:
+    """Remove falsy values from a list."""
+    return [v for v in value if v]
```

Comment:
that need tests

Reply:
Do you mean unit tests or test if a dictionary?